### PR TITLE
creating test for text and date columns

### PR DESCRIPTION
**Pull request by javidlc**
@AnaMJaimeR and @felipemonroy please review this PR. Some important things:
- for text I did not add the empty and null test because when tried uploading empty and null columns they are converted to float. But, because we specified the date columns, in the date test i added.
- for the date test, math.isna is not working for NaT. So i used pd.isna
- with error = coerce, empty, null, spaces and all incorrect date format is converted to null. Therefore, they are not unique values
- please confirm the new bugfixes before tried to run the unit test

### CODE REVIEW

File: src/test/test_datetime.py [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+        # Condition test
+        self.assert_nan = False
+
+        # Expected
+        self.expected_name = "this_is_the_serie_name"
+        self.expected_get_unique = 4
+        self.expected_get_missing = 4
+        self.expected_get_weekend = 3
+        self.expected_get_weekday = 2
+        self.expected_get_future = 1
```
[the remark this answers is not in the document]
Comment (javidlc) [reply src/test/test_datetime.py:251]: done!

File: src/test/test_text.py [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+                AXIS_FONT_SIZE=14,
+                TICK_FONT_SIZE=16,
+                CATEGORY_ORDER="total descending",
+                TEMPLATE="simple_white",
+                TITLE="title",
+            ),
+        )
+
+        # Condition test
+        # self.assert_nan = False
```
[the remark this answers is not in the document]
Comment (javidlc) [reply src/test/test_text.py:238]: done

File: src/test/test_datetime.py [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+        self.assertListEqual(result.columns.tolist(), expected_columns)
+        # Assert: length
+        self.assertEqual(len(result), expected_length)
+
+
+class TestMethodsValidDateColumn(TestMethodsDate, unittest.TestCase):
+    """Class containing the setup and customised tests for the scenario of valid serie as input."""
+
+    def setUp(self) -> None:
+
```
[the remark this answers is not in the document]
Comment (javidlc) [reply src/test/test_datetime.py:209]: done

File: src/test/test_text.py [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+        self.expected_get_digit = 1
+        self.expected_get_mode = "cc"
+        self.expected_length = 5
+
+    def tearDown(self) -> None:
+        del (
+            self.serie,
+            self.text_class,
+            self.params,
+            # self.assert_nan,
```
[the remark this answers is not in the document]
Comment (javidlc) [reply src/test/test_text.py:258]: done

File: src/test/test_text.py [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+        self.assertListEqual(result.columns.tolist(), expected_columns)
+        # Assert: length
+        self.assertEqual(len(result), expected_length)
+
+
+class TestMethodsValidTextColumn(TestMethodsText, unittest.TestCase):
+    """Class containing the setup and customised tests for the scenario of valid serie as input."""
+
+    def setUp(self) -> None:
+
```
[the remark this answers is not in the document]
Comment (javidlc) [reply src/test/test_text.py:204]: done

**Review by felipemonroy — APPROVED** [review]
